### PR TITLE
[doc] Add a small note about the interconnect

### PR DESCRIPTION
**Pull request by a-will**
Add a short summary of top_earlgrey's interconnect topology and the
access latency.

Signed-off-by: Alexander Williams <awill@google.com>

### CODE REVIEW

File: hw/top_earlgrey/doc/_index.md [hunk 1]
```diff
@@ -449,6 +449,21 @@ For more details see the [flash controller module specification]({{< relref "hw/
...
 See the [OpenTitan bus specification]({{< relref "hw/ip/tlul/doc" >}}) for more details.
 
+#### Topology
+`top_earlgrey` has a two-level hierarchy for its bus network.
+Close to the CPU is the high-speed cluster, with low access latency.
+Farther from the CPU, through a low-speed bridge, is the low-speed cluster, with higher access latency.
+
+#### Performance
+
+The following table describes the typical number of CPU cycles from a transaction's launch to its completion for a device in the specified cluster:
```
Comment (rswarbrick) [inline hw/top_earlgrey/doc/_index.md:459]: Maybe worth adding a caveat about the bus not being busy?

**Review by rswarbrick — APPROVED** [review]

File: hw/top_earlgrey/doc/_index.md (hunk 1 above)
Comment (a-will) [reply hw/top_earlgrey/doc/_index.md]: Done. :)

File: hw/top_earlgrey/doc/_index.md [hunk 2]
```diff
@@ -449,6 +449,22 @@ For more details see the [flash controller module specification]({{< relref "hw/
...
 
+#### Topology
+`top_earlgrey` has a two-level hierarchy for its bus network.
+Close to the CPU is the high-speed cluster, with low access latency.
+Farther from the CPU, through a low-speed bridge, is the low-speed cluster, with higher access latency.
+
+#### Performance
+
+The following table describes the typical number of CPU cycles from a transaction's launch to its completion for a device in the specified cluster.
+Note that these values assume there is no bus contention.
```
Comment (tjaychen) [inline hw/top_earlgrey/doc/_index.md:460]: heh, it helps that we only have one host in the system.. although i suppose it's possible for instructions to contend with data on the rom / flash.

File: hw/top_earlgrey/doc/_index.md [hunk 3]
```diff
@@ -449,6 +449,22 @@ For more details see the [flash controller module specification]({{< relref "hw/
 Interconnecting the processor and peripheral and memory units is a bus network built upon the TileLink-Uncached-Light protocol.
 See the [OpenTitan bus specification]({{< relref "hw/ip/tlul/doc" >}}) for more details.
 
+#### Topology
+`top_earlgrey` has a two-level hierarchy for its bus network.
+Close to the CPU is the high-speed cluster, with low access latency.
+Farther from the CPU, through a low-speed bridge, is the low-speed cluster, with higher access latency.
```
Comment (tjaychen) [inline hw/top_earlgrey/doc/_index.md:455]: i'll add a diagram to this area later.

**Review by tjaychen — APPROVED** [review]

**Pull request merged**